format/spec.md

```diff
@@ -1297,54 +1308,56 @@ Example
+| **`unknown`**        | Not supported                                                                                                |
+| **`boolean`**        | `0x00` for false, non-zero byte for true                                                                     |
+| **`int`**            | Stored as 4-byte little-endian                                                                               |
+| **`long`**           | Stored as 8-byte little-endian                                                                               |
+| **`float`**          | Stored as 4-byte little-endian                                                                               |
+| **`double`**         | Stored as 8-byte little-endian                                                                               |
+| **`date`**           | Stores days from the 1970-01-01 in an 4-byte little-endian int                                               |
+| **`time`**           | Stores microseconds from midnight in an 8-byte little-endian long                                            |
+| **`timestamp`**      | Stores microseconds from 1970-01-01 00:00:00.000000 in an 8-byte little-endian long                          |
+| **`timestamptz`**    | Stores microseconds from 1970-01-01 00:00:00.000000 UTC in an 8-byte little-endian long                      |
+| **`timestamp_ns`**   | Stores nanoseconds from 1970-01-01 00:00:00.000000000 in an 8-byte little-endian long                        |
+| **`timestamptz_ns`** | Stores nanoseconds from 1970-01-01 00:00:00.000000000 UTC in an 8-byte little-endian long                    |
+| **`string`**         | UTF-8 bytes (without length)                                                                                 |
+| **`uuid`**           | 16-byte big-endian value, see example in Appendix B                                                          |
+| **`fixed(L)`**       | Binary value                                                                                                 |
+| **`binary`**         | Binary value (without length)                                                                                |
+| **`decimal(P, S)`**  | Stores unscaled value as two’s-complement big-endian binary, using the minimum number of bytes for the value |
+| **`struct`**         | Not supported                                                                                                |
+| **`list`**           | Not supported                                                                                                |
+| **`map`**            | Not supported                                                                                                |
+| **`variant`**        | Not supported                                                                                                |
```

Comment:
@RussellSpitzer, didn't we want to say that this should be a Variant `value` that contains a Variant of a value for each shredded column? I don't want to miss this in v3 or else we won't be able to do file skipping.

Reply:
We will need to specify how to handle the `metadata` (concat?)

Reply:
Yes I agree, but currently we don't have any info about shredding so I didn't want to include it yet